task-force-docs/buildings/overture_facadeMaterial_OSM_mapping.json

```diff
@@ -0,0 +1,86 @@
+{
+  "brick": [
+    "brick",
+    "slate",
+    "ladrillos",
+    "masonry",
+    "silicate brick",
+    "bricks",
+    "unburnt brick",
+    "silicate_brick",
+    "brick_block",
+    "muddy_brick"
+  ],
+  "cementBlock": [
+    "cement_block",
+    "block",
+    "cement block",
+    "cement_blocks",
+    "cement blocks"
+  ],
+  "clay": [
+    "clay",
+    "mud",
+    "rammed_earth",
+    "loam",
+    "earth",
+    "grass",
+    "pressed_soil_blocks"
+  ],
+  "concrete": [
+    "concrete",
+    "concrete masonry unit",
+    "cement",
+    "concrete_reinforced",
+    "reinforced_concrete"
+  ],
+  "glass": [
+    "glass",
+    "mirror"
+  ],
+  "metal": [
+    "metal",
+    "steel",
+    "metal_plates",
+    "tin",
+    "iron_sheet",
+    "copper",
+    "metal_sheet",
+    "ironsheets",
+    "panel",
+    "aluminium"
+  ],
+  "plaster": [
+    "plaster",
+    "plastered",
+    "hard"
+  ],
+  "plastic": [
+    "plastic",
+    "vinyl",
+    "plastic_sheeting",
+    "composite",
+    "vinyl_siding"
+  ],
+  "stone": [
+    "stone",
+    "limestone",
+    "sandstone",
+    "granite",
+    "marble",
+    "tiles"
+  ],
+  "timberFraming": [
+    "timber_framing",
+    "traditional"
+  ],
+  "wood": [
+    "wood",
+    "reed",
+    "wattle_and_daub",
+    "timber_planks",
+    "wood/masonry",
+    "wood/bamboo",
+    "bamboo"
+  ]
+}
```

Comment:
@LukaszMichalski-TomTom could you fix your tooling setup so it adds a newline at the end of the files?